src/measure/Measure.i

```diff
@@ -96,6 +96,55 @@
 };
 
 #if defined SWIGRUBY
+  %feature("director:except") {
+    // Mimic openstudio::evalString for a possible approach to reporting more info or a callstack.
+    VALUE errinfo = rb_errinfo();
+    VALUE exception_class = rb_obj_class(errinfo);
+    VALUE classNameValue = rb_class_name(exception_class);
+    std::string className(StringValuePtr(classNameValue));
+
+    VALUE errstr = rb_obj_as_string(errinfo);
+    std::string errMessage(StringValuePtr(errstr));
+
+    std::string totalErr = className + ": " + errMessage;
+
+    // I just **cannot** figure out a way to get the error location in C, without calling $@.to_s. Seems nothing is available here
+    int locationError;
+    VALUE locval = rb_eval_string_protect("$@.to_s", &locationError);
+    std::string loc;
+    if (locationError == 0) {
+      loc = StringValuePtr(locval);
+    } else {
+      loc = "Failed to get VM location";
+    }
```

Comment:
This bit seems redundant with the backtrace block below, you get the same info twice.